src/plugins/discover/public/utils/format_hit.test.ts

```diff
@@ -49,7 +49,6 @@ describe('formatHit', () => {
       ['message', 'formatted:foobar'],
       ['object.value', 'formatted:42,13'],
       ['_index', 'formatted:logs'],
-      ['_score', undefined],
```

Comment:
note: this PR is removing `_score` if no value is set, which is a better behavior than the existing one